### DISCUSSION TITLE
tutorials/free/top/examples

### DISCUSSION THREAD

**Discussion by giscus** [Stuyk/altv-athena#406] [Documentation Feedback]
# tutorials/free/top/examples

Roleplay Script for GTA:V

https://athenaframework.com/tutorials/free/top/examples

**Comment by OlegGulevskyy** [comment]
Awesome job! Y
ou have sections Jobs and Prompt for User Input hidden under section Custom Storage Solutions. Probably not what you intended?

**Reply by Stuyk** [reply]
Thanks. I'll fix it.
